docs/src/gui/qtvcp.adoc

```diff
@@ -669,176 +868,208 @@ def get_handlers(halcomp,widgets,paths):
-There is a button in the top tool bar of designer for editing signals.
-After pushing it, if you click-and-hold on the button it will show a arrow
-(looks like a ground signal from electrical schematic)
-Slide this arrow to a part of the main window that does not have widgets on it.
-A 'Configure Connections' dialog will pop up.
-The list on the left are the available signals from the widget.
-The list on the right is the available slots on the main window and you can add to it.
-
-Pick the signal 'clicked()' - this makes the slots side available.
-click 'edit' on the slots list.
-A 'Slots/Signals of MainWindow' dialog will pop up.
-On the slots list at the top there is a plus icon - click it.
-you can now edit a new slot name.
-Erase the default name 'slot()' and change it to test_button()
-press the ok button.
-You'll be back to the 'Configure Connections' dialog.
-now you can select your new slot in the slot list.
-then press ok and save the file.
-
-.Designer signal/slot selection
-image::images/designer_slots.png["QTvcp",align="left"]
-
-=== Handler file changes
-
-Now you must add the function to the handler file.
-The function signature is 'def slotname(self):'
-We will add some code to print the widget name.
-
-So for our example:
+*In the Qt Designer editor*:
+
+* You _create user function slots_
+* You _connect the slots to widgets using signals_.
+
+*In the handler file*:
+
+* You _create the slot's functions_ defined in Qt Designer.
+
+[[sub:qtvcp:designer-slots]]
+=== Using Qt Designer To Add Slots
+
+When you have loaded your screen into Qt Designer, add a plain
+`PushButton` to the screen. +
+You could change the name of the button to something interesting like
+'test_button'.
+
+There are two ways to edit connections - This is the graphical way.
+
+* There is a button in the top tool bar of Qt Designer for editing signals.
+  After pushing it, if you click-and-hold on the button it will show an
+  arrow (looks like a ground signal from electrical schematic).
+* Slide this arrow to a part of the main window that does not have widgets
+  on it.
+* A 'Configure Connections' dialog will pop up.
+** The list on the left are the available signals from the widget.
+** The list on the right are the available slots on the main window and
+   you can add to it.
+* Pick the signal `clicked()` - this makes the slots side available.
+* Click 'Edit' on the slots list.
+* A 'Slots/Signals of MainWindow' dialog will pop up.
+* On the slots list at the top there is a '+' icon - click it.
+* You can now edit a new slot name.
+* Erase the default name `slot()` and change it to `test_button()`
+* Press the 'OK' button.
+* You'll be back to the 'Configure Connections' dialog.
+* Now you can select your new slot in the slot list.
+* Then press 'OK' and save the file.
+
+.Qt Designer Signal/Slot Selection
+image::images/designer_slots.png["Qt Designer Signal/Slot Selection",align="center"]
+
+=== Handler File Changes
```

Comment:
"Handler File Changes"

Maybe here can be found a better heading? This heading is more confusing than helpful IMHO.

Reply:
Handler file changes are those changes that are performed by the handler to files. File Handler changes are changes performed to the file handler. Just keep it like the original and accept, I suggest.

Reply:
Yeah just want top improve the understanding of the original...